install/rdtclient-install.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2023 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y unzip
+msg_ok "Installed Dependencies"
+
+msg_info "Installing ASP.NET Core Runtime"
+$STD bash -c "$(wget --no-cache -qLO - https://dot.net/v1/dotnet-install.sh)"
```

Comment:
What led you to choose this option instead of the `.deb` package?

Reply:
Two things:
First, since you previously asked why I was using a ubuntu package on debian, I just went to RTFM on how to properly install this dependency on debian.
Second, on debian 12 there was a package not found using the previous option

Reply:
`wget https://packages.microsoft.com/config/debian/12/packages-microsoft-prod.deb` is working.
Whenever possible, I strive to minimize the use of external bash scripts, as it simplifies the process of due diligence. For instance, the `dotnet-install.sh` script, with its 1746 lines of code, can be quite challenging to thoroughly examine.

Reply:
The https://github.com/tteck/Proxmox/blob/main/install/technitiumdns-install.sh script utilizes that `.deb` (Debian 11)

Reply:
I understand your POV with the script and I tend to agree

just spin up a debian 12 lxc, and this is the result:

root@debian:~# apt-get update
(...)
root@debian:~# apt-get install dotnet-sdk-6.0
(...)
E: Unable to locate package dotnet-sdk-6.0

I can go with the debian 11 package since it works. Up to you